packages/web-components/src/components/masthead/__stories__/README.stories.react.mdx

```diff
@@ -248,19 +284,45 @@ The API results must match the following structure:
   ['result 1', 'result 2', 'result 3'],
```

Comment:
Same comment here as above, do we have a type for these API results that we could point people to in the preceeding sentance?